network-api/networkapi/wagtailpages/templates/wagtailpages/pages/youtube-regrets-2022/fragments/we_found_section.html

```diff
@@ -0,0 +1,21 @@
+{% load i18n %}
+<section class="tw-dark tw-my-7 large:tw-my-[120px]">
+  <div class="container tw-max-w-3xl tw-mx-auto">
+    <h4 class="tw-text-lg medium:tw-text-[1.375rem] tw-uppercase tw-font-sans tw-font-bold tw-mb-5">{% trans 'Ultimately, we found:' %}</h4>
```

Comment:
Question From the place in the document, should this maybe be a h3 instead?

Reply:
Yes good point i'll change that!